src/services/tech-record-document-conversion.ts

```diff
@@ -241,6 +243,24 @@ const upsertVehicle = async (
   connection: Connection,
   techRecordDocument: TechRecordDocument,
 ): Promise<number> => {
+
+  if (techRecordDocument.techRecord && techRecordDocument.techRecord[0].statusCode === 'archived') {
+      debugLog('upsertTechRecords: Retrieving vehicle...');
+      // retrieve records associated with systemNumber
+      const existingRecordIds = await selectRecordIds(
+          VEHICLE_TABLE.tableName,
+          { system_number: techRecordDocument.systemNumber, vin: vinCleanser(techRecordDocument.vin) },
+          connection,
+      );
+      if (existingRecordIds.rows.length > 0) {
+          return existingRecordIds.rows[0].id;
+      } else {
+          debugLog(
```

Comment:
Would be useful to have this at the info level

Reply:
It looks like the repo leans more towards only console logging errors, all console.info logs are wrapped in the debugLog, probably to avoid spamming CW. So I'm not sure there is a need for us to deviate from that. 

If we want the logging we can quite easily turn it on.